public/js/chat-cmds.js

```diff
@@ -141,15 +92,46 @@ var ListUser = function() {
     return this;
 };
 
-var _toggleCmdHelp = function(_this, name) {
+/**
+ * Toggles command help.
+ *
+ * @param {Object} toggle - The toggle element (_e.g._ Camera, Microphone)
+ * for which we want help.
+ * @param {String} name - The name of the toggle element.
+ * @returns {Function} Function that returns command help as an HTML string.
+ * @private
+ */
+var _toggleCmdHelp = function(toggle, name) {
+    /**
+     * Returns command help as an HTML string.
+     *
+     * @returns {String} Command help.
+     */
     return function() {
-        return '<h2><span class="chatIntCmdName">/' + _this.command + '</span> ' +
+        return '<h2><span class="chatIntCmdName">/' + toggle.command + '</span> ' +
```

Comment:
`_this` is usually reserved for aliasing `this`, rather than as a function param